wgengine/router/ifconfig_windows.go

```diff
@@ -91,6 +91,7 @@ func monitorDefaultRoutes(device *device.Device, autoMTU bool, tun *tun.NativeTu
 		}
 		err = bindSocketRoute(winipcfg.AF_INET6, device, ourLuid, &lastLuid6)
 		if err != nil {
+			log.Printf("bindSocketRoute(AF_INET6): %v", err)
```

Comment:
Remove?

Reply:
This doesn't fire right now (at least on my machine). But we might be getting lucky because of the two IPv6 routes I had/have. I want to keep this in so if it happens to other people we know.